### PR TITLE
Release N51

### PR DESCRIPTION
**Pull request by janhohenheim**
_Resolves #1505_

[Rendered](https://github.com/rust-gamedev/rust-gamedev.github.io/blob/release-51/content/news/051/index.md)

Depends on #1520. The link-checking CI step will fail until it is merged first.

### CODE REVIEW

File: content/news/051/index.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-draft = true
+draft = false
 +++
 
 <!-- no toc -->
 
 <!-- Check the post with markdownlint-->
 
-Welcome to the 51th issue of the Rust GameDev Workgroup's
+Welcome to the 51st issue of the Rust GameDev Workgroup's
```
Comment (Vrixyz) [inline content/news/051/index.md:12]: good catch!

File: content/news/051/index.md [hunk 2]
```diff
@@ -279,32 +313,32 @@ Special mention to the following talks, relevant to game development:
...
+- [Hanabi 0.11](https://github.com/djeedai/bevy_hanabi/blob/v0.11.0/CHANGELOG.md): A GPU particle system plugin for the Bevy game engine.
 
-<!-- Links to "good first issue"-labels or direct links to specific tasks -->
+### Blog Posts
 
-## Jobs
-
-<!-- An optional section for new jobs related to Rust gamedev -->
+- [Architectural Ruminations][pixel-wizards-arch]: reflections on using Rust for game development,
+ prompted by LogLog Games's _[Leaving Rust game dev after 3 years]_.
```
Comment (Vrixyz) [inline content/news/051/index.md:335]: I'm not totally opposed to this, but it's listed in 2 places (the other being in pixel-wizards section)

**Review by Vrixyz — COMMENTED** [review]
LGTM :)

File: content/news/051/index.md (hunk 1 above)
Comment (janhohenheim) [reply content/news/051/index.md:12]: :D

File: content/news/051/index.md (hunk 2 above)
Comment (janhohenheim) [reply content/news/051/index.md]: You're right. Fixed.

File: content/news/051/bevy_rep.png
Comment (AngelOnFira) [inline content/news/051/bevy_rep.png:1]: Can we reduce the size of this image?

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
Just one item for now, I'll try to do a pass on it later myself :)

File: content/news/051/bevy_rep.png
Comment (janhohenheim) [reply content/news/051/bevy_rep.png:1]: Good catch, my bad. It's at 255 KiB now.

**Review by AngelOnFira — APPROVED** [review]
Looks good, just need to update the date :)

**Pull request merged**